#!/usr/bin/env python
PACKAGE = "lla2enu"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("lower_limit",    int_t,    0, "lower limit", 1,  0, 25)
gen.add("higher_limit", int_t, 1, "higher limit",    5, 1,   50)

exit(gen.generate(PACKAGE, "distance_cal", "parameters"))
